src/rating-calculator/record-comparator.ts

```diff
@@ -56,6 +56,18 @@ export function compareSongsByNextRating(
   );
 }
 
+export function compareSongsByNewRating(
+    record1: ChartRecordWithRating,
+    record2: ChartRecordWithRating
+) {
+  const newRating1 = Math.floor(record1.rating) + record1.nextRanks.values().next().value;
+  const newRating2 = Math.floor(record2.rating) + record2.nextRanks.values().next().value;
```

Comment:
This formula (= rating + nextRank) is incorrect for the same reason I described in the other comment.

Reply:
No longer required